### PR TITLE
Updates to VLASIATOR_ARCH files

### PR DESCRIPTION
**Pull request by MiroPalmu**
Updated the instructions for fixing broken VLSV files for Ukko and Carrington make arches.

Also added make arch and script to install dependencies for Cubbli 20, which should work for any local machine but cubbli 20 is only one that is tested. 

In the script I chose to install the libs to folder located at root folder of Vlasiator even if in other arch files libraries are usually located at ~/libs because I don't like that scripts from git repo modify files outside of the repo and anyway I think there really small change of sharing these libraries with other programs so there is no benefit to install them to $(HOME)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ nbproject
 vlasiator
 vlasiator+pat
 nbproject
+external_libs/
```
Comment (ykempf) [inline .gitignore:14]: Better add a newline here.

File: MAKE/Makefile.cubbli20 [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+INC_BOOST = -I/usr/include/boost/
+LIB_BOOST = -L/usr/lib/x86_64-linux-gnu/ -lboost_program_options
+
+
+# In case of there is need for optional libs Silo and Jmalloc
+# you can install them manually and add path here 
+
+# Optional
+INC_SILO = #-I$(LIBRARY_PREFIX)/include
+LIB_SILO = #-L$(LIBRARY_PREFIX)/lib -lsilo
```
Comment (ykempf) [inline MAKE/Makefile.cubbli20:36]: SILO can be removed altogether from here I think.

File: MAKE/configure_cubbli20.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+)
+                                                                                                                      
+    # Vectorclass/version1
+(
+    cd $EXTERNAL_LIBS_FOLDER
+    git clone https://github.com/vectorclass/version1
+    git clone https://github.com/vectorclass/add-on
+    cp add-on/vector3d/vector3d.h version1
+    rm -rf add-on
+)
```
Comment (ykempf) [inline MAKE/configure_cubbli20.sh:72]: Here too add a newline please.

**Review by ykempf — CHANGES_REQUESTED** [review]
Well now that there's also a new feature coming in I'll wait for @ursg's opinion before we merge.

**Review by ursg — APPROVED** [review]
Looks all reasonable. Not sure about that .gitignore addition, but otherwise w could merge this right away.

Watch out that the changes to the carrington makfiles might cause a merge conflict with #629, but that will be pretty easy to resolve, I suppose.

File: MAKE/Makefile.cubbli20 [hunk 4]
```diff
@@ -0,0 +1,117 @@
+# This is meant for building local copy of Vlasiator.
+# It is tested with cubbli 20 but it should work on other systems as well
+
+# This is a modified version of Makefile.yann
+
+# There is script called configure_cubbli20.sh that will install the dependencies automatically.
```
[the remark this answers is not in the document]
Comment (henry2004y) [reply MAKE/Makefile.cubbli20:6]: Miro used shell script for this, which I guess is more universal. I have some extra functionalities in my Python script like checking if any of the dependencies has been installed, etc. The tricky part is the validity of the download links.

It's great to have an automatic script for installation, especially locally.

[the remark this answers is not in the document]
Comment (MiroPalmu) [reply MAKE/Makefile.cubbli20:6]: I tried to keep the script simple and easy to read what it is going to do.

My thought process was to just put commands from [here](https://github.com/fmihpc/vlasiator/wiki/Installing-Vlasiator) to one script that just installs the libraries and nothing else. (Also fix some of the outdated links etc.)

File: .gitignore [hunk 5]
```diff
@@ -11,3 +11,7 @@ nbproject
...
+
+
+
```
[the remark this answers is not in the document]
Comment (MiroPalmu) [reply .gitignore]: I removed .gitignore from the PR

**Pull request merged**